### DISCUSSION TITLE
Password storage support?

### DISCUSSION THREAD

**Discussion by 1c383r9** [niceit/tabby-cloud-sync-settings#33] [Q&A] [answered]
Luv this plugin...It makes tabby more versatile, and I sync it on gist.

Lately found that the password couldn't be sync'ed. Shall I turn on the vault or something? I use basic password auth and a bit of private key.

Kudos.

**Comment by niceit** [comment, marked as answer]
hello @2ndMessiah the password is not saved into the config file by default, that is restricted by the Tabby application, not the plugin.
